packages/conf-llvm/conf-llvm.15.0.7/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "Kate <kit.ty.kate@disroot.org>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build: [
+  ["bash" "-ex" "configure.sh" version]
+]
+depends: [
+  "conf-bash" {build}
+]
+depexts: [
+  ["llvm@15"] {os-distribution = "homebrew" & os = "macos"}
```

Comment:
The `macos-homebrew-*-arm64` CI jobs fail with `ld: library not found for -lzstd` but the llvm@15 homebrew package does require zstd, so the failure is odd. I don't know if there is something nonstandard about the CI worker setup.